### PR TITLE
wso2-ei: simplified kubernetes setup 

### PR DESCRIPTION
**Pull request by NishikaDeSilva**

### CODE REVIEW

File: simple/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* An already setup Kubernetes cluster. If you are unfamiliar with this context, you can use [this guide](https://kubernetes.io/docs/setup/pick-right-solution/) to set up the cluster.
+* WSO2 subscribed users can run **wso2ei.sh** with latest updates by providing their subscription username and password. If you do not possess an active WSO2 subscription already, run **wso2ei_open.sh** which does not require subscription credentials.
+
+## Quick Start Guide
+1. Download simplified kubernetes setup for WSO2 Enterprise Integrator (either **wso2ei.sh** or **wso2ei_open.sh**).From this point forward the steps are being described for wso2ei.sh. If you have downloaded wso2ei_open.sh please substitute wso2ei.sh with wso2ei_open.sh for every command.
+
+1. In the command line, move into the directory where you have downloaded the simplified kubernetes-apim setup. (Usually, the file would be in the “Downloads” directory unless you have changed the default directory to somewhere else.)
+1. Provide permissions for the setup file to execute by running **chmod +x wso2ei.sh**
```
Comment (nihlaakram) [inline simple/README.md]: All points are numbered as 1. Shall we fix this?

File: simple/wso2ei.sh [hunk 2]
```diff
@@ -0,0 +1,3067 @@
...
+NODE_IP=''
+
+#bash functions
+function usage(){
+  echo "Usage: "
+  echo -e "-d, --deploy     Deploy WSO2 Enterprise Integrator"
+  echo -e "-u, --undeploy   Undeploy WSO2 Enterprise Integrator"
+  echo -e "-h, --help       Display usage instrusctions"
+}
+function undeploy(){
```
Comment (nihlaakram) [inline simple/wso2ei.sh]: Some functions have a new line space before and some don't . Shall we fix this?

File: simple/README.md (hunk 1 above)
Comment (NishikaDeSilva) [reply simple/README.md]: Screen Shot 2019-04-29 at 2 17 09 PM
this is how it looks at the end. (a Numbered list)

File: simple/wso2ei.sh (hunk 2 above)
Comment (NishikaDeSilva) [reply simple/wso2ei.sh]: ok. what is the best practice here? To have a newline or not?

File: simple/README.md (hunk 1 above)
Comment (nihlaakram) [reply simple/README.md]: Please refer https://help.github.com/en/articles/basic-writing-and-formatting-syntax#lists for odered list syntax.

Comment (NishikaDeSilva) [reply simple/README.md]: will do

File: simple/wso2ei.sh (hunk 2 above)
Comment (Savidude) [reply simple/wso2ei.sh]: Best practice is to have an empty line between functions, mainly for the purpose of readability.

**Pull request closed without merging**